modules/gapi/include/opencv2/gapi/gproto.hpp

```diff
@@ -76,6 +76,23 @@ template<typename... Ts> inline GProtoOutputArgs GOut(Ts&&... ts)
     return GProtoOutputArgs(detail::packArgs(std::forward<Ts>(ts)...));
 }
 
+// Extract elements form tuple
+template<typename... Ts, int... Indexes>
+static GProtoOutputArgs getGOut_impl(std::tuple<Ts...>& ts, detail::Seq<Indexes...>)
+{
+    return GProtoOutputArgs{ detail::packArgs(std::get<Indexes>(ts)...)};
+}
```

Comment:
This function pollutes `cv::` namespace now - and so, the documentation.
At least, please move it to `cv::detail` namespace.
Please also put a "FIXME: Someday utilize a generic tuple_to_vec<> routine" comment.